frontend/src/app/components/op-context-menu/op-context-menu-handler.ts

```diff
@@ -18,11 +18,11 @@ export abstract class OpContextMenuHandler extends UntilDestroyedMixin {
    * Called when the service closes this context menu
    */
   public onClose() {
-    this.afterFocusOn.focus();
+    this.afterFocusOn.trigger('focus');
   }
 
   public onOpen(menu:JQuery) {
-    menu.find('.menu-item').first().focus();
+    menu.find('.menu-item').first().trigger('focus');
```

Comment:
this is a drive-by fix - i noticed that `focus` is a deprecated method while reading some menu handler code. The deprecation notice says to use `trigger()` instead.